05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,4 +1,20 @@
-.lawn {
+.container {
+  width:100%;
+}
+.berryimage{
+  width:500px;
+  float:left;
+}
+/*.berryimage img{
+  width:500px;
+}*/
+.berrycontent{
+  width:500px;
+  float:left;
```

Comment:
if you have both berryimage and berrycontent have the same width and float, could you factor those properties out?